### ISSUE TITLE
Jottacloud non standard mount point doesn’t seem to work if device or mountpoint has space in it

### ISSUE THREAD

**Issue by martingjohn** [rclone/rclone#4697]
#### What is the problem you are having with rclone?
I'm trying to access the "Backed Up" folders in Jottacloud using the non standard mount point option. This works ok for ones without spaces in but errors when the computer backed up has a space in it (or if the mountpoint has a space in). The config script seems to find the device ok and puts it in the config file
```
Do you want to use a non standard device/mountpoint e.g. for accessing files uploaded using the official Jottacloud client?

y) Yes
n) No (default)
y/n> y
Please select the device to use. Normally this will be Jotta
Choose a number from below, or type in an existing value
 1 > gui-vm
 2 > Jotta
 3 > Martin MacBook Air
 4 > MSI-laptop
 5 > nas1
 6 > PCS desktop
 7 > WIN2008
 8 > WINDOWS10
Devices> 3
Please select the mountpoint to user. Normally this will be Archive
Choose a number from below, or type in an existing value
 1 > Desktop
 2 > Documents
 3 > Downloads
 4 > Interface
 5 > Pictures
 6 > WTF
Mountpoints> 1
```
or
```
Do you want to use a non standard device/mountpoint e.g. for accessing files uploaded using the official Jottacloud client?

y) Yes
n) No (default)
y/n> y
Please select the device to use. Normally this will be Jotta
Choose a number from below, or type in an existing value
 1 > gui-vm
 2 > Jotta
 3 > Martin MacBook Air
 4 > MSI-laptop
 5 > nas1
 6 > PCS desktop
 7 > WIN2008
 8 > WINDOWS10
Devices> 7
Please select the mountpoint to user. Normally this will be Archive
Choose a number from below, or type in an existing value
 1 > Desktop
 2 > Downloads
 3 > My Documents
 4 > My Pictures
 5 > W
 6 > X
Mountpoints> 3
```

#### What is your rclone version (output from `rclone version`)
Tried with
```
rclone v1.53.1
- os/arch: freebsd/amd64
- go version: go1.15
```
and
```
rclone v1.54.0-beta.4831.4e2b5389d
- os/arch: freebsd/amd64
- go version: go1.15.2
```

#### Which OS you are using and how many bits (eg Windows 7, 64 bit)
FreeBSD, 64 bit

####  Which cloud storage system are you using? (eg Google Drive)
Jottacloud

#### The command you were trying to run (eg `rclone copy /tmp remote:tmp`)
```
rclone lsd jc_test:
```

#### The rclone config contents with secrets removed.
```
[jc_test]
type = jottacloud
tokenURL = https://id.jottacloud.com/auth/realms/jottacloud/protocol/openid-connect/token
token = {"access_token":"secret","token_type":"bearer","refresh_token":"secret","expiry":"2020-10-19T13:11:09.390448678+01:00"}
device = Martin MacBook Air
mountpoint = Desktop
configVersion = 1
```
or
```
[jc_test]
type = jottacloud
tokenURL = https://id.jottacloud.com/auth/realms/jottacloud/protocol/openid-connect/token
token = {"access_token":"secret","token_type":"bearer","refresh_token":"secret","expiry":"2020-10-19T13:11:09.390448678+01:00"}
device = WIN2008
mountpoint = My Documents
configVersion = 1
```
#### A log from the command with the `-vv` flag (eg output from `rclone -vv copy /tmp remote:tmp`)
```
2020/10/19 12:27:25 DEBUG : rclone: Version "v1.54.0-beta.4831.4e2b5389d" starting with parameters ["rclone" "lsd" "-vv" "jc_test:"]
2020/10/19 12:27:25 DEBUG : Using config file from "/root/.config/rclone/rclone.conf"
2020/10/19 12:27:25 DEBUG : Creating backend with remote "jc_test:"
2020/10/19 12:27:26 ERROR : : error listing: directory not found
2020/10/19 12:27:26 DEBUG : 5 go routines active
2020/10/19 12:27:26 Failed to lsd with 2 errors: last error was: directory not found
```

**Comment by albertony** [comment]
I gave this a quick debugging, and seems this line should not have the urlPathEscape:

https://github.com/rclone/rclone/blob/4e2b5389d75fbe245330a70adcf4911a63299d55/backend/jottacloud/jottacloud.go#L556

I can test it a bit more, and submit a pull request (if not someone else beats me to it).

**Comment by ncw** [comment]
That was quick @albertony :-)

I've pushed this to a branch so it will make a beta - @martingjohn  can you check out this in 20 minutes when it has build?

https://beta.rclone.org/branch/pr-4698-jottacloud/v1.54.0-beta.4832.d3c026fb7.pr-4698-jottacloud/

**Comment by martingjohn** [comment]
Wow, amazing work guys - that was super fast.

I can confirm that now works with both devices with spaces in and mountpoints with spaces in

**Comment by ncw** [comment]
I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org) in 15-30 mins and released in v1.54 and also in v1.53.2
